grpc-gcp/src/test/java/com/google/grpc/gcp/SpannerIntegrationTest.java

```diff
@@ -299,39 +409,39 @@ public void testExecuteSqlAsync() throws Exception {
     for (String respName : respNames) {
       AsyncResponseObserver<PartialResultSet> resp = new AsyncResponseObserver<>();
       stub.executeStreamingSql(
-          ExecuteSqlRequest.newBuilder().setSession(respName).setSql("select * FROM jenny").build(),
+          ExecuteSqlRequest.newBuilder().setSession(respName).setSql("select * FROM Users").build(),
           resp);
-      assertEquals(USERNAME, resp.get().getValues(0).getStringValue());
+      assertEquals(USERNAME, resp.get().getValues(1).getStringValue());
     }
     deleteAsyncSessions(stub, respNames);
   }
 
   @Test
   public void testBoundWithInvalidAffinityKey() throws Exception {
     SpannerBlockingStub stub = getSpannerBlockingStub();
-    CreateSessionRequest req = CreateSessionRequest.newBuilder().setDatabase(DATABASE).build();
+    CreateSessionRequest req = CreateSessionRequest.newBuilder().setDatabase(DATABASE_PATH).build();
     Session session = stub.createSession(req);
     expectedEx.expect(StatusRuntimeException.class);
     expectedEx.expectMessage("INVALID_ARGUMENT: Invalid GetSession request.");
-    stub.getSession(GetSessionRequest.newBuilder().setName("jennnny").build());
+    stub.getSession(GetSessionRequest.newBuilder().setName("invalid_session").build());
     deleteSession(stub, session);
```

Comment:
deleting could be done in tearDown() method?
so that test methods could just have the test specific logic.

Reply:
Thanks. There are already logic for cleaning up sessions, I will remove those duplicated operations.